scss/_patterns_grid.scss

```
@@ -190,40 +191,51 @@
   }
 
   // Common grid patterns
+  $col-50-medium: $grid-columns-medium / 2;
```

This throws a warning when building sass, should be wrapped in `calc`
```suggestion
  $col-50-medium: calc($grid-columns-medium / 2);
```